### PR TITLE
Add shows

### LINKED ISSUE #9: Add Shows (Tv and Movie)
=>Form Design 
=>Api call to add movies

### PR DESCRIPTION
**Pull request by shailja-imw**
=>Form design
=> API call
=> on success  message

closes #9

### CODE REVIEW

**Comment by shailja-imw** [comment]
image

File: app/pages/add.vue [hunk 1]
```diff
@@ -1,3 +1,54 @@
...
+  }
+}
+</script>
+
 <template>
-  <div>from /add the movies/tv-shows</div>
+  <div class="container">
+    <h3 class="text-center my-2">Add New Shows</h3>
+    <FormShows
+      ref="addUpdateForm"
```
Comment (monikagorkhiya) [inline app/pages/add.vue:49]: why need ref?

File: app/components/FormShows.vue [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+const imgUrl = ref(props.showDeatils.imgUrl)
+const productionCountry = ref(props.showDeatils.productionCountr)
+const role = ref(props.showDeatils.credits.role)
+const name = ref(props.showDeatils.credits.name)
+const img_url = ref(props.showDeatils.credits.img_url)
+const character = ref(props.showDeatils.credits.character)
+let details = {
+  title: title.value,
+  description: description.value,
+  type: type.value,
```
Comment (monikagorkhiya) [inline app/components/FormShows.vue:18]: Can we reduce the code here?

File: app/components/FormShows.vue [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+      title_id: title.value,
+      name: name.value,
+      character: character.value,
+      role: role.value,
+      img_url: img_url.value,
+    },
+  ],
+}
+watchEffect(() => {
+  details = {
```
Comment (monikagorkhiya) [inline app/components/FormShows.vue:35]: Can we reduce the code here?

File: app/components/FormShows.vue [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+        role: role.value,
+        img_url: img_url.value,
+      },
+    ],
+  }
+})
+const emit = defineEmits(['handleSubmit'])
+
+function submitForm() {
+  console.log(details)
```
Comment (monikagorkhiya) [inline app/components/FormShows.vue:58]: remove console.log

**Review by monikagorkhiya — CHANGES_REQUESTED** [review]

File: app/pages/add.vue (hunk 1 above)
Comment (shailja-imw) [reply app/pages/add.vue:49]: no need of the ref I was trying something else and forgot to remove the ref. I will remove it.

File: app/components/FormShows.vue (hunk 2 above)
Comment (shailja-imw) [reply app/components/FormShows.vue:18]: yes I am trying for the alternative ways to reduce the code and optimise it

File: app/components/FormShows.vue (hunk 4 above)
Comment (shailja-imw) [reply app/components/FormShows.vue:58]: ok

**Pull request merged**